common/domain/replication_queue.go

```diff
@@ -336,13 +336,14 @@ func (q *replicationQueueImpl) emitDLQSize() {
 				).IncCounter(
 					metrics.DomainReplicationQueueSizeErrorCount,
 				)
+			} else {
+				q.metricsClient.Scope(
```

Comment:
is there the corresponding metric on the dash?

Reply:
Yes